lemmatized_text/models.py

```diff
@@ -146,14 +146,14 @@ def clone_url(self):
         return f"{url}?cloned_from={self.pk}"
 
     def clone(self, cloned_by=None):
-        return LemmatizedText.objects.create(
-            title=self.title,
-            lang=self.lang,
-            original_text=self.original_text,
-            cloned_from=self,
-            data=self.data,
-            created_by=cloned_by or self.created_by,
-        )
+        """Set a copy of this object's pk to None, set some relationships and save (cloning)"""
+        # https://docs.djangoproject.com/en/3.2/topics/db/queries/#copying-model-instances
+        obj = LemmatizedText.objects.get(pk=self.pk)  # declaring 'self' here results in a ValueError
+        obj.pk = None
+        obj.cloned_from = self
+        obj.created_by = cloned_by or self.created_by
+        obj.save()
```

Comment:
Just an observation but should we change the title or have an indicator that the text is a clone?
Everything Looks great Tylor!

Reply:
I think it's a good idea, but I hesitate to programmatically edit any user provided content. But there should be some way to more easily distinguish between original text and cloned text. I think the best UX will shake out after some use.